### ISSUE TITLE
potential misconfiguration of APC and IFV, regarding vehicle racks

### ISSUE THREAD

**Issue by kOepi123** [IDI-Systems/acre2#471]
**Arma 3 Version:** stable
**CBA Version:** `3.x.x` stable
**ACRE2 Version:** `2.x.x` dev .981

**Mods:**
```
- CBA_A3
- acre2
- RHSRF

**Description:**
- only the gunner position has access to the vehiclerack

**Steps to reproduce:**
- editor, singleplayer. put an btr60 and an bmp1 into the map, mount them as driver, gunner, commander. only gunner can open the radio to make changes.

- expected: only commander can make changes

**Where did the issue occur?**
- Editor (Singleplayer)

**Placed Modules:**
- none

**Comment by TheMagnetar** [comment]
It is not within my scope, or that of ACRE, to provide full compatibility with other mods. However, I will investigate if it is a configuration issue, but my guess is that custom RHS classes are causing compatibility issues.

**Comment by jonpas** [comment]
We will **not be providing compatibility for other mods** within ACRE itself. Ideally other mods will add compatibility themselves or accept it from the rest of the community (that is also easiest).

However we are aware that some projects simply refuse to do that (for whatever reason), so if someone wishes to maintain a compatibility mod, we are willing to host it under IDI-Systems organization and considered "official compatibility mod maintained by ACRE community".

Mind though, those mods would **not be maintained by us**, someone else from the community would have to do that and also handle releases, we are not capable of maintaining much else at this time, we can only offer hosting, tools, build server (automated releases are possible) and help.

**Comment by kOepi123** [comment]
yea, not talking about compatibility or maintaining anything. I stumbled over something and with my thinking it is a misconfiguration. Since I dont think anyone would want it to be this way, it must be a mistake. Can it be a misconfiguration by RHS? yes. Is it a misconfiguration by ACRE? should be easy to find out for the involved to double check it.
The wiki said that all children of x and y dont need to be configured, I assumed that these vehicles are children, because you can use the vehicle racks in them.

**Comment by jonpas** [comment]
Depends from what they inherit specifically, we can only make sure it works for vanilla vehicles correctly, it's all inheritance from there. As we modify some vanilla vehicle specifically, anything inheriting from that would obviously get same configuration. I have no clue how RHS is setup, can you check their `AcreRacks` config specifically and show us that?

**Comment by kOepi123** [comment]
I did not intend to track down the problem, I just wanted to help make sure it is not on acre side. I myself have no clue where to find the acre configs from RHS, if there are any. Which I doubt that there are, thats why I posted on here. I will have to make a custom vehicle rack config for my project or I can let magnetar know about it. When he confirms that it is not an RHS issue, I saved myelf some work and helped improve acre.
